src/python/tests/core/issue_management/monorail/monorail_test.py

```diff
@@ -210,5 +217,53 @@ def test_modify_components(self):
                           self.itm.last_issue.components)
 
   def test_get_original_issue(self):
+    """Test get_original_issue."""
     issue = self.issue_tracker.get_original_issue(1338)
     self.assertEqual(1337, issue.id)
+
+  def test_find_issues(self):
+    """Test find_issues."""
+    issue0 = MonorailIssue()
+    issue0.id = 1
+
+    issue1 = MonorailIssue()
+    issue1.id = 2
+    self.mock.get_issues.return_value = [
+        issue0,
+        issue1,
+    ]
+    issues = self.issue_tracker.find_issues(
+        keywords=['one', 'two'], only_open=True)
+    self.assertItemsEqual([1, 2], [issue.id for issue in issues])
+
+    self.mock.get_issues.assert_has_calls([
+        mock.call(mock.ANY, '"one" "two"', can='open'),
+    ])
+
+    issues = self.issue_tracker.find_issues(
+        keywords=['one', 'two'], only_open=False)
+    self.assertItemsEqual([1, 2], [issue.id for issue in issues])
+
+    self.mock.get_issues.assert_has_calls([
+        mock.call(mock.ANY, '"one" "two"', can='all'),
+    ])
+
+  def test_find_issues_url(self):
+    """Test find_issues_url."""
+    url = self.issue_tracker.find_issues_url(
+        keywords=['one', 'two'], only_open=False)
+    self.assertEqual(
+        'https://bugs.chromium.org/p/name/issues/list'
+        '?can_id=1&q=%22one%22+%22two%22', url)
+
+    url = self.issue_tracker.find_issues_url(
+        keywords=['one', 'two'], only_open=True)
+    self.assertEqual(
+        'https://bugs.chromium.org/p/name/issues/list'
+        '?can_id=2&q=%22one%22+%22two%22', url)
+
+  def test_issue_url(self):
+    """Test issue_url."""
+    issue_url = self.issue_tracker.issue_url(1337)
+    self.assertEqual('https://bugs.chromium.org/p/name/issues/detail?id=1337',
```

Comment:
nit: instead of name, we should use like project or project_name ?

Reply:
Done!